dataset: scan	No info from HuggingFace
dataset: scan,simple	splits: train: 16728 rows, test: 4182 rows
dataset: scan,simple	split: train	columns: commands, actions
dataset: scan,addprim_jump	splits: train: 14670 rows, test: 7706 rows
dataset: scan,addprim_jump	split: train	columns: commands, actions
dataset: scan,addprim_turn_left	splits: train: 21890 rows, test: 1208 rows
dataset: scan,addprim_turn_left	split: train	columns: commands, actions
dataset: scan,filler_num0	splits: train: 15225 rows, test: 1173 rows
dataset: scan,filler_num0	split: train	columns: commands, actions
dataset: scan,filler_num1	splits: train: 16290 rows, test: 1173 rows
dataset: scan,filler_num1	split: train	columns: commands, actions
dataset: scan,filler_num2	splits: train: 17391 rows, test: 1173 rows
dataset: scan,filler_num2	split: train	columns: commands, actions
dataset: scan,filler_num3	splits: train: 18528 rows, test: 1173 rows
dataset: scan,filler_num3	split: train	columns: commands, actions
dataset: scan,length	splits: train: 16990 rows, test: 3920 rows
dataset: scan,length	split: train	columns: commands, actions
dataset: scan,template_around_right	splits: train: 15225 rows, test: 4476 rows
dataset: scan,template_around_right	split: train	columns: commands, actions
dataset: scan,template_jump_around_right	splits: train: 18528 rows, test: 1173 rows
dataset: scan,template_jump_around_right	split: train	columns: commands, actions
dataset: scan,template_opposite_right	splits: train: 15225 rows, test: 4476 rows
dataset: scan,template_opposite_right	split: train	columns: commands, actions
dataset: scan,template_right	splits: train: 15225 rows, test: 4476 rows
dataset: scan,template_right	split: train	columns: commands, actions
***ERROR***
model: None
data: H:scan
eqn: None
splits: train
split: train
task: None
error: (<class 'requests.exceptions.HTTPError'>, HTTPError('405 Client Error: Not Allowed for url: https://huggingface.co/api/models-tags-by-type'), <traceback object at 0x7f73e9f77140>)
